tutorials/how_to/elmo.md

```diff
@@ -53,10 +52,31 @@ JSON-serialized string with a mapping from sentences to line indices to the
 
 For more details, see `allennlp elmo -h`. 
 
-## Using ELMo programmatically
+## Using ELMo interactively
+
+You can use ELMo interactively (or programatically) with iPython.  The `allennlp.commands.elmo.ElmoEmbedder` class provides the easiest way to process one or many sentences with ELMo, but it is meant for use as a standalone command and not within a larger model.  For example, if you would like to learn a weighted average of the ELMo vectors then you need to use `allennlp.modules.elmo.Elmo` instead.
+
+The `ElmoEmbedder` command returns three vectors, each corresponding to a layer in the ELMo LSTM output.  The first layer captures mostly syntactic information and the last layer captures more of the semantics.
```

Comment:
Replace this with "The `ElmoEmbedder` class returns three vectors for each word, each vector corresponding to a layer in the ELMo LSTM output.  The first layer corresponds to the context insensitive token representation, followed by the two LSTM layers.  See [the ELMo paper](http://www.aclweb.org/anthology/N18-1202) or [follow up work at EMNLP 2018](https://arxiv.org/abs/1808.08949) for a description of what types of information is captured in each layer."